client/blocks/inline-help/popover.jsx

```diff
@@ -133,6 +133,7 @@ class InlineHelpPopover extends Component {
 		if ( this.props.isOnboardingWelcomeVisible ) {
 			return this.openOnboardingWelcomeView();
 		}
+		this.props.hideChecklistPrompt();
```

Comment:
We'll need to import the `hideChecklistPrompt` action and bind it to the props. Getting this error:

https://cldup.com/xEXA_f0lvn.mp4

Reply:
Fixed! Thanks for pointing it out.